ansible-satellite6/satellite.yml

```diff
@@ -0,0 +1,5 @@
+---
+- hosts: all
+  remote_user: root
```

Comment:
I think it can be better to avoid setting this in the playbook and rather set that at the inventory level.